cmk/gui/plugins/wato/notifications.py

```diff
@@ -396,6 +396,28 @@ def spec(self):
         )
 
 
+@notification_parameter_registry.register
+class NotificationParameterSIGNL4(NotificationParameter):
+    @property
+    def ident(self):
+        return "signl4"
+
+    @property
+    def spec(self):
+        return Dictionary(
+            title=_("Create notification with the following parameters"),
+            optional_keys = [],
+            elements = [
+                ("password", Password(
+                    title = _("Team Secret"),
```

Comment:
You could alternatively implement a cascading dropdown to give the user a choice if he/she wants to write the team key directly or use a defined from a password store. As an example you can take a look at the implementation of slack.